orca-core/src/main/groovy/com/netflix/spinnaker/orca/batch/ExecutionPropagationListener.groovy

```diff
@@ -87,7 +87,7 @@ class ExecutionPropagationListener extends JobExecutionListenerSupport implement
       jobExecution.exitStatus = ExitStatus.STOPPED
     }
 
-    if (orcaTaskStatus == STOPPED) {
+    if (orcaTaskStatus?.isSuccessful()) {
```

Comment:
suspect we would need to modify echo to bypass skipped stages as well. https://github.com/spinnaker/echo/blob/78500ab1bff77bf1dc1072e52bd00c2cada64043/echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/AbstractEventNotificationAgent.groovy#L70-L72